docs/securing_fledge.rst

```diff
@@ -349,23 +349,32 @@ Default ca certificate is available inside $FLEDGE_DATA/etc/certs and named as c
 
 Below are the steps to create custom certificate along with existing fledge based ca signed for auth certificates.
 
-a) Create a new certificate for username. Let say **test**
+**Using cURL**
 
-.. code-block:: console
+- To create a user with a unique name and case-insensitive. The cURL commands are provided below.
+
+.. note::
+
+    You need administrator privileges to perform this action.
 
-    $ cd $FLEDGE_ROOT
-    $ ./scripts/auth_certificates user test 365
+For example, Add a username with the name **test**
 
-    Here script arguments are: $1=user $2=FLEDGE_USERNAME $3=SSL_DAYS_EXPIRATION
+.. code-block:: console
 
-And now you can find **test** cert inside $FLEDGE_DATA/etc/certs/
+    $ AUTH_TOKEN=$(curl -d '{"username": "<ADMIN_USERNAME>", "password": "<ADMIN_PASSWORD>"}' -sX POST <PROTOCOL>://<FLEDGE_IP>:<FLEDGE_REST_API_PORT>/fledge/login | jq '.token' | tr -d '""')
+    $ USER_ID=$(curl -H "authorization: $AUTH_TOKEN" -skX POST <PROTOCOL>://<FLEDGE_IP>:<FLEDGE_REST_API_PORT>/fledge/admin/user -d '{"username":"test","real_name":"Test","access_method":"cert","description":"Non-admin based role","role_id":2}' | jq '.user.userId')
 
-b) Now, it's time to create user with name **test** (case sensitive). Also only admin can create user. Below are the cURL Commands
+- It is now time to generate a new certificate for the **test** username that was created earlier.
 
 .. code-block:: console
 
-    $ AUTH_TOKEN=$(curl -d '{"username": "admin", "password": "fledge"}' -sX POST <PROTOCOL>://<FLEDGE_IP>:<FLEDGE_REST_API_PORT>/fledge/login | jq '.token' | tr -d '""')
-    $ curl -H "authorization: $AUTH_TOKEN" -skX POST <PROTOCOL>://<FLEDGE_IP>:<FLEDGE_REST_API_PORT>/fledge/admin/user -d '{"username":"test","real_name":"Test","access_method":"cert","description":"Non-admin based role","role_id":2}'
+    $ curl -H "authorization: $AUTH_TOKEN" -sX POST <PROTOCOL>://<FLEDGE_IP>:<FLEDGE_REST_API_PORT>/fledge/admin/$USER_ID/authcertificate
+
+You can now locate the **test** certificate within the $FLEDGE_DATA/etc/certs/ and can be used for login.
+
+.. warning::
+
+    For security concerns, it is highly recommended to download or copy the certificate and delete it from the certificate store. Relocate it to a secure, private location to prevent unauthorized access and sharing, using encrypted storage or password protection.
```

Comment:
We should probably be more explicit about which field to download. It might also be worth considering an API to do this, otherwise we rely upon users having access to the Linux machine to download the certificate.